src/Discord.Net.Core/Entities/Activities/SpotifyGame.cs

```diff
@@ -31,6 +31,23 @@ public class SpotifyGame : Game
         ///     A string containing the name of the song (e.g. <c>Lonely Together (feat. Rita Ora)</c>).
         /// </returns>
         public string TrackTitle { get; internal set; }
+
+        /// <summary>
+        ///     Gets the date the track began playing at.
+        /// </summary>
+        /// <returns>
+        ///     A <see cref="DateTimeOffset"/> containing the start timestamp of the song.
+        /// </returns>
+        public DateTimeOffset? StartedAt { get; internal set; }
+
+        /// <summary>
+        ///     Gets the date the track will stop playing at.
```

Comment:
`Gets the date when the track ends.`

Reply:
https://github.com/discord-net/Discord.Net/pull/1414/commits/2f9e286c09116022692d7344227c708815d4bbfb